ocfweb/docs/docs/services/lab/printing.md

```diff
@@ -27,7 +27,7 @@ Check your print balance regularly!
 ## Printing refunds
 
 Pages are **not** normally refunded, except in cases of technical issues. In
-this case, you should talk with the Publications Center staff at the front desk
+this case, you should talk with the OCF Operations Staff at the front desk
```

Comment:
This isn't actually correct though, right? Like they can't give refunds currently

Reply:
technically not lmao but at least they can direct you to get refunds or they can print the thing for you

Reply:
Yeah, less inaccurate than it was before at least